src/desktop/bundled/kis_tablet/pch.h

```diff
@@ -0,0 +1,10 @@
+#ifdef _WIN32
```

Comment:
Why is this Windows-specific stuff needed in this one? Also, I am surprised there is a measurable performance benefit to doing anything special here since this library consists of only two objects. How much time is this saving, do you know?